<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnlineMode</class>
 <widget class="QWidget" name="OnlineMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select service</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="youtubeBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Youtube</string>
     </property>
     <property name="icon">
      <iconset resource="res/resource.qrc">
       <normaloff>:/icons/services_youtube.png</normaloff>:/icons/services_youtube.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="spotifyBtn">
     <property name="text">
      <string>Spotify</string>
     </property>
     <property name="icon">
      <iconset resource="res/resource.qrc">
       <normaloff>:/icons/service_spotify.png</normaloff>:/icons/service_spotify.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="soundcloudBtn">
     <property name="text">
      <string>SoundCloud</string>
     </property>
     <property name="icon">
      <iconset resource="res/resource.qrc">
       <normaloff>:/icons/service_soundcloud.png</normaloff>:/icons/service_soundcloud.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/resource.qrc"/>
 </resources>
 <connections/>
</ui>
